catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -79,9 +87,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SchematronValidationService.class);
 
-    private TransformerFactory transformerFactory;
+    public static final String DEFAULT_THREAD_POOL_SIZE = "128";
```

Comment:
I still think this is too large a default and that we should set a lower cap on the number of threads in the exec pool.